conf/routes

```diff
@@ -19,8 +19,13 @@ POST    /roles/update                                       controllers.Roles.up
 
 # Rest api
 GET     /rest/profile                                       controllers.RestApi.profile
-POST    /rest/users                                         controllers.RestApi.users
-POST    /rest/users/:id                                     controllers.RestApi.user(id : String)
+POST    /rest/users/create                                  controllers.RestApi.createUser
+PUT    /rest/users/avatar                                   controllers.RestApi.updateUserProfileImage
+POST    /rest/users                                         controllers.RestApi.getUsers
+POST    /rest/users/:id                                     controllers.RestApi.getUser(id : String)
+PUT     /rest/users                                         controllers.RestApi.updateUser
```

Comment:
ID? :)

Reply:
i used the email as identifier. This will be transmited in the request body.
If we prefere IDs in the URL, i will refactor

Reply:
We will use the id in the uri and the email in the body as identifier. 
this should prevent, that users are deleted by mistake.